src/javascript/utils/helper.js

```diff
@@ -5,3 +5,45 @@ export const truncate = (string, num) => {
 
     return string.slice(0, num) + '...';
 };
+
+/**
+ * JCR system name doesnt allow this characters: ['/', ':', '[', ']', '|', '*']
```

Comment:
Do we accept the '\' character?

Reply:
yes, invalid character are from that list:
https://docs.adobe.com/docs/en/spec/jcr/2.0/3_Repository_Model.html#3.2.2%20Local%20Names